microcosm_flask/fields/query_string_list.py

```diff
@@ -22,7 +27,7 @@ def _deserialize(self, value, attr, obj):
 
         try:
             attribute_elements = [attr_element.split(",") for attr_element in obj.getlist(attr)]
-            attribute_params = [param for attr_param in attribute_elements for param in attr_param]
+            attribute_params = SelfSerializableList(param for attr_param in attribute_elements for param in attr_param)
```

Comment:
Two thoughts here:
 1. The naming of the list could be better; something like `PrintableList` might be enough; I don't think "Self" is useful or clarifying
 2. Does the output here need to be a list of typed-values (e.g. enums)? Could we avoid the extra class and just use:
```
[str(item) for item in items]
```